Use Qt defined type quint16 instead of GNU libc
internal type __uint16_t.

--- ksmserver/fadeeffect.cpp	2015-05-07 20:13:26.000000000 +0200
+++ ksmserver/fadeeffect.cpp	2015-10-09 00:29:17.537504092 +0200
@@ -237,7 +237,7 @@
                          __m128i *result, uint length)
 {
     length = (length + 15) >> 4;
-    register const __m128i alpha    = _mm_set1_epi16(__uint16_t (a));
+    register const __m128i alpha    = _mm_set1_epi16(quint16 (a));
     register const __m128i negalpha = _mm_xor_si128(alpha, _mm_set1_epi16 (0x00ff));
 
     for (uint i = 0; i < length; i++)
